helpers/index.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @namespace helpers
+ * @property {module:helpers/bson} BSON
+ * @property {module:helpers/cache} Cache
+ * @property {module:helpers/config} Config
+ * @property {module:helpers/diff} Diff
+ * @property {module:helpers/ed} Ed
+ * @property {module:helpers/git} Git
+ * @property {module:helpers/http_api} HTTP_API
+ * @property {module:helpers/jobs_queue} Jobs_queue
+ * @property {module:helpers/slots} Slots
+ * @property {module:helpers/sort_by} Sort_by
+ * @property {module:helpers/swagger} Swagger
+ * @property {module:helpers/swagger_module_registry} Swagger_module_registry
+ * @property {module:helpers/ws_api} WS_API
+ * @property {module:helpers/z_schema} Z_schema
+ */
```

Comment:
This list doesn't include all the files. What were the criteria for inclusion? (It also affects what shows up in the HTML documentation.)

Reply:
It only includes the modules. 
Classes and namespaces can have a `@memberof parent` tag, which will include them to the parent.